src/Presentation/Nop.Web/Views/Shared/Components/SocialButtons/Default.cshtml

```diff
@@ -2,18 +2,18 @@
 <ul class="networks">
 	@if (!string.IsNullOrEmpty(Model.FacebookLink))
 	{
-		<li class="facebook"><a href="@Model.FacebookLink" target="_blank">@T("Footer.FollowUs.Facebook")</a></li>
+		<li class="facebook"><a href="@Model.FacebookLink" target="_blank" aria-label="@T("Footer.FollowUs.Facebook")">@T("Footer.FollowUs.Facebook")</a></li>
```

Comment:
Could you please share the tool you used? I'm interested why we shoud add aria-label here if this link is self-described

Reply:
Yes we use Lighthouse Accessibility scan in Google Chrome, SortSite and some of our clients use AChecker.

Reply:
But what about the links? why shoud we add aria-label here if this link is self-described? I'm interested which tool shows the error here. I tried to use tools for Google Chrome but it don't show any issues with these links

Reply:
I see what you are asking now. I will remove this from the commit. I realize that the issue is more from Nop Templates and not from Nop Default Clean, and Nop Templates has agreed to resolve this in their themes where they did not previously have any text for the link.